Signal/src/view controllers/MessagesViewController.m

```diff
@@ -1609,8 +1610,6 @@ - (void)yapDatabaseModified:(NSNotification *)notification {
       for (YapDatabaseViewRowChange *rowChange in messageRowChanges) {
           switch (rowChange.type) {
               case YapDatabaseViewChangeDelete: {
-                  TSInteraction *interaction = [self interactionAtIndexPath:rowChange.indexPath];
```

Comment:
What happens if a deletion happens that is triggered by some other code than the one above? I think it should still be supported.